docs/core/store.md

```diff
@@ -222,6 +222,24 @@ When `Store.{Get, Set}()` is called, the store forwards the call to its parent,
 
 When `Store.Iterator()` is called, it does not simply prefix the `Store.prefix`, since it does not work as intended. In that case, some of the elements are traversed even they are not starting with the prefix.
 
+## Table Store
```

Comment:
I added this section both in `docs/core/store.md` and `store/README.md`. In follow-up PRs, when adding more advanced features, I'm thinking about having a dedicated `spec` folder in `store/table` to have more detailed documentation.
cc/ @ryanchristo